js/main.js

```diff
@@ -4,9 +4,32 @@
 var $photoUrl = document.querySelector('#imgURL');
 var $img = document.querySelector('img');
 var $form = document.querySelector('.newEntry');
+var $newEntryView = document.querySelector('.new-button');
+var $navEntries = document.querySelector('.nav-entries');
+var $views = document.querySelectorAll('.view');
+var $ul = document.querySelector('.journal-entries');
 
 $photoUrl.addEventListener('input', handleImgUrl);
 $form.addEventListener('submit', handleSubmit);
+$newEntryView.addEventListener('click', showNewForm);
+$navEntries.addEventListener('click', showEntries);
+
+if (!data.entries.length) {
+  $ul.textContent = 'No entries have been recorded.';
```

Comment:
Instead of setting the text content of the `ul` like this (`ul` elements should never have text content immediately inside of them), you should have a text element such as a h1-h6 or p element that you hide and show as necessary